src/main/java/org/broadinstitute/hellbender/utils/RandomDNA.java

```diff
@@ -1,90 +0,0 @@
-package org.broadinstitute.hellbender.utils;
-
-import java.util.Random;
-
-/**
- * Random DNA sequence generator.
- *
- * <p>
- *     Returned bases are always in upper case and one of the valid four nocleotides 'A', 'C', 'G' and 'T'.
- * </p>
- *
- * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
- */
-public final class RandomDNA {
```

Comment:
I think that the `RandomDNA` could be useful for the testing framework packaged into the main jar (the same as other test classes)...

Reply:
Talked to our engine guys and  currently unless we know that someone it is using it downstream is best to remove it from the main jar. It seems that the fact that there are some test classes in the main is not ideal but result of dependencies issues when we had the project split in public and protected. 

I suspect that eventually we may create a separate artifact (a test jar) for those classes.

Reply:
I think that this utilities could still leave in the main jar, under the `utils.test` package. The artifact with them is still not done and it may prevent downstream projects to use this useful testing classes for their own purpose. I've already requested this at some point to avoid packaging also TestNG and other testing dependencies, but removing completely some testing framework until it is package properly sounds like a bad idea to re-use the code in the GATK-framework...